### DISCUSSION TITLE
讨论频道使用说明

### DISCUSSION THREAD

**Discussion by Dabaiee** [SutuLabs/zkcourse#2] [公告频道]
欢迎来到ZK Shanghai Workshop讨论频道，本频道主要用于集中解答同学们上课、完成作业遇到的问题。
为了营造更有效率的学习空间，也可以更多地帮助其他人，建议如下
1. 遇到困难，寻求帮助时：先在Discussion讨论频道查询是否有同学提过相同相似的问题，可以寻找解答或者力所能及地帮助其他人。
2. 如果没有相关的解答，可以在Week1- Week8 频道里，按照课程对应的阶段发起新的discussion提问
3. 在讨论频道发出问题的同时，也在所在的学习小组提问

**Comment by tomyoung-dhu** [comment]
你好请问怎样可以加入这个workshop

**Reply by DessertHeart** [reply]
https://zkshanghai.xyz/24fall/syllabus.html

可留意

**Reply by tomyoung-dhu** [reply]
好的，感谢🙏
